permissions/plugin-pipeline-timeline.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-timeline"
+github: "jenkinsci/pipeline-timeline-plugin"
+paths:
+- "org/jenkins-ci/plugins/pipeline-timeline"
```

Comment:
This is wrong. Should be `"io/jenkins/plugins/pipeline-timeline"`
https://github.com/jenkinsci/pipeline-timeline-plugin/blob/15163f02711c5fbd30009ae652585407b4809258/pom.xml#L10-L11

Reply:
Thanks for reviewing!

The correction has been made as of https://github.com/jenkins-infra/repository-permissions-updater/pull/979/commits/c62117e13f40a56446e711e4b92e974c5ed3bdcf.